src/Utils/RemovableInputField.js

```diff
@@ -1,16 +1,29 @@
 import React, { Component } from 'react';
 
-class InputField extends Component {
+class RemovableInputField extends Component {
     constructor(props) {
         super(props);
         //Function binding
         this.handleInput = this.handleInput.bind(this);
+        this.handleRemove = this.handleRemove.bind(this);
     }
 
     handleInput(event) {
         this.props.onInputChange(this.props.id, event.target.value);
     }
 
+    handleRemove(event) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "event". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)